### PR TITLE
add ip to pod wide output

### PR DESCRIPTION
**Pull request by adohe-zz**
## Pull Request Guidelines
1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
3. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.

``` release-note
* Use the release-note-* labels to set the release note state 
* Clear this block to use the PR title as the release note 
-OR-
* Enter your extended release note here
```

[Analytics]()

### CODE REVIEW

**Comment by AdoHe** [comment]
Please refer #21381 for more detail. @bgrant0607 @smarterclayton ptal.

**Comment by bgrant0607** [comment]
@AdoHe Thanks.

Please update the PR description.

File: pkg/kubectl/resource_printer.go [hunk 1]
```diff
@@ -633,8 +633,13 @@ func printPodBase(pod *api.Pod, w io.Writer, options PrintOptions) error {
 
 	if options.Wide {
 		nodeName := pod.Spec.NodeName
-		if _, err := fmt.Fprintf(w, "\t%s",
+		podIP := pod.Status.PodIP
```
Comment (bgrant0607) [inline pkg/kubectl/resource_printer.go]: I'd prefer to put IP before NODE. It will usually be shorter.

**Comment by bgrant0607** [comment]
cc @kubernetes/kubectl

**Comment by AdoHe** [comment]
@janetkuo ptal.

**Pull request merged**